File: storydoc-frontend/src/app/api/models/code-trace-dto.ts
/* tslint:disable */
/* eslint-disable */
import { SourceCodeConfigCoordinate } from './source-code-config-coordinate';
import { StitchItemDto } from './stitch-item-dto';
import { StoryDocSummaryDto } from './story-doc-summary-dto';
export interface CodeTraceDto {
  config?: SourceCodeConfigCoordinate;
  description?: string;
  items?: Array<StitchItemDto>;
  name?: string;
  storyDocSummary?: StoryDocSummaryDto;
}

File: storydoc-backend/src/main/java/io/storydoc/server/code/app/stitch/CodeTraceDTO.java
package io.storydoc.server.code.app.stitch;

import io.storydoc.server.code.domain.SourceCodeConfigCoordinate;
import io.storydoc.server.code.domain.StitchConfigCoordinate;
import io.storydoc.server.storydoc.app.dto.StoryDocSummaryDTO;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
@Builder
public class CodeTraceDTO {
    private String description;
    private StoryDocSummaryDTO storyDocSummary;
    private List<StitchItemDTO> items;
    private String name;
    public SourceCodeConfigCoordinate config;
    public StitchConfigCoordinate stitchConfigCoordinate;
}
